README.md

```diff
@@ -503,7 +536,66 @@ sudo docker run --rm \
     simplelogin/app:3.2.2 python init_app.py
 ```
 
-Now, it's time to run the `webapp` container!
+now it time to setup the `webapp` and `email handler` containers
+
+for docker-compose add this to your file:
+
+```
+    simplelogin:
+        image: simplelogin/app:3.2.2
+        container_name: sl-app
+        ports:
+            - "7777:7777"
+        restart: unless-stopped
+        volumes:
+        # this is to add more words to create more unique aliases
+        # - ./words.txt:/code/local_data/words.txt
+        - ./sl:/sl
+        - ./sl/upload:/code/static/upload
+        - ./simplelogin.env:/code/.env
+        - ./dkim.key:/dkim.key
+        - ./dkim.pub.key:/dkim.pub.key
+
+    simplelogin-email_handler:
+        image: simplelogin/app:3.2.2
+        container_name: sl-email
+        command: python email_handler.py
+        ports:
+            - "20381:20381"
+        restart: unless-stopped
+        volumes:
+        # this is to add more words to create more unique aliases you have to create a word.txt file.
+        # - ./words.txt:/code/local_data/words.txt
+        - ./sl:/sl
+        - ./sl/upload:/code/static/upload
+        - ./simplelogin.env:/code/.env
+        - ./dkim.key:/dkim.key
+        - ./dkim.pub.key:/dkim.pub.key
+
+```
+
+**note**: the words.txt volume is optional and it allows you to add your own custom words for random aliases.
```

Comment:
To keep the self-hosting instructions small, I think it's better to move this section to a dedicated file in the `docs/` folder.

Reply:
alright we can create like a new page for more stuff after initial setup

it should also include how create ssl certificates since its not **essential** for the app to work

Reply:
fixed https://github.com/simple-login/app/pull/309/commits/b374f5620091577d57b5d04322aaaf76aa291ad6